src/__tests__/Search/SearchResultsList.test.tsx

```diff
@@ -85,22 +86,23 @@ describe('SearchResultsList', () => {
   });
 
   it('should select the new revision and uncheck the previous one when clicking a different base revision', async () => {
-    // set delay to null to prevent test time-out due to useFakeTimers
     const user = userEvent.setup({ delay: null });
-
     await renderComponent();
-    // focus input to show results
```

Comment:
You should add the removed comments back.

Reply:
definitely